doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -686,6 +686,12 @@ Common Application Framework (CAF)
 
   * Reduced verbosity of logs denoting allowed power states from ``info`` to ``debug``.
 
+* :ref:`caf_shell`:
+
+  * Added a new module ``caf_shell`` as a shell interface for triggering CAF events.
+  * Added the :kconfig:option:`CONFIG_CAF_SHELL` Kconfig option for enabling the module.
+  * Added a shell command for submitting CAF events, and a subcommand for button event submission.
```

Comment:
```suggestion
* Added :ref:`caf_shell` for triggering CAF events.
```
As this is a totally new module, no need to say more at this point. Suggesting that you move this right under the Common Application Framework CAF heading.